web/src/components/netflow-table/netflow-table-row.tsx

```diff
@@ -64,6 +76,16 @@ const NetflowTableRow: React.FC<{ flow: Record; columns: Column[]; size: Size }>
           return '';
         }
       }
+      case ColumnsId.srcwkd:
+      case ColumnsId.dstwkd: {
+        const nsValue = c.id === ColumnsId.srcwkd ? flow.labels.SrcNamespace : flow.labels.DstNamespace;
+        const kind = c.id === ColumnsId.srcwkd ? flow.fields.SrcWorkloadKind : flow.fields.DstWorkloadKind;
+        if (value && kind && nsValue) {
+          return <ResourceLink kind={kind} name={value.toString()} namespace={nsValue.toString()} />;
+        } else {
+          return '';
+        }
+      }
```

Comment:
Tooltip is missing here

You can also merge these case with `srcpod` / `dstpod` since it's the same ui